extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -87,7 +105,7 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     this.mutateElement(() => {
       createShadowRootWithStyle(
         this.element,
-        renderShoppingTagTemplate(tagData),
+        renderShoppingTagTemplate(tagData, this.onClick_.bind(this)),
```

Comment:
This works but `.bind` is old-school, lets use the new school syntax and use fat arrows to scope this :)
This can work something like:
`renderShoppingTagTemplate(tagData, (tagData) => this.onClick_(tagData))`
Please see comment on line 84 and 35 for context on why `tagData` is being passed through and why it's beneficial.